### ISSUE TITLE
InfluxDB 2.x support

### ISSUE THREAD

**Issue by maxadamo** [chengshiwen/influxdb-cluster#14]
__Proposal:__ is there any plan to support InfluxDB 2.x ?

**Comment by chengshiwen** [comment]
InfluxDB Enterprise 2.x has not been released yet, and the next plan for InfluxDB Cluster is to adapt to [InfluxDB Enterprise 1.9.x](https://docs.influxdata.com/enterprise_influxdb/v1.9/) and [InfluxDB Enterprise 1.10.x](https://docs.influxdata.com/enterprise_influxdb/v1.10/)

**Comment by maxadamo** [comment]
hi @chengshiwen , I understand what you mean, but I belive that they will never release InfluxDB cluster 2.x. 
IMO cluster 2.x already exists, but it's available only on the cloud (though they won't disclose the details, and they'll only tell you that they guarantee 99% uptime)  
I wonder if you can apply the existings replication mechanism to InfluxDB 2.x OpenSource.

**Comment by maxadamo** [comment]
p.s.: I'm an enterprise customer and I could try to raise it with them once again, but I expect them to be slippery and evasive as usual.

**Comment by chengshiwen** [comment]
>  I believe that there will never be a public release of InfluxDB cluster 2.x.

Maybe so. I will release v1.8.10-c1.2.0 as soon as next month to support [anti-entropy](https://docs.influxdata.com/enterprise_influxdb/v1.8/administration/anti-entropy/) feature. Then I will try to figure out the clustering mechanism of InfluxDB Cloud, which requires some work.

**Comment by MasterCATZ** [comment]
hopefully v2 is not too far away 

will there be any integrated auto incremental backup  scripts ?

**Comment by chengshiwen** [comment]
> will there be any integrated auto incremental backup scripts ?

Yes. InfluxDB Cluster v1.8.10-c1.2.0 will support (incremental) backup and restore like the official document:
- https://docs.influxdata.com/enterprise_influxdb/v1.8/administration/backup-and-restore/#backup-utility
- https://docs.influxdata.com/enterprise_influxdb/v1.8/administration/backup-and-restore/#restore-utility

> and any idea what I can do to change this from a docker container to a LXC to run with proxmox ?

I will try to give a solution on the new patch version v1.8.10-c1.1.2, which will be released in the last two weeks

**Comment by MasterCATZ** [comment]
ok thanks , I will give this a spin , I have  a docker lxc running to run this docker container inside off anyhow , hopefully Home Assistant will not complain about  downgrading the  influxdb version

**Comment by 593769290** [comment]
> 

when will v1.8.10-c1.2.0 be release ?

**Comment by 593769290** [comment]
> @MasterCATZ
> 
> > will there be any integrated auto incremental backup scripts ?
> 
> Yes. InfluxDB Cluster v1.8.10-c1.2.0 will support (incremental) backup and restore like the official document:
> 
> * https://docs.influxdata.com/enterprise_influxdb/v1.8/administration/backup-and-restore/#backup-utility
> * https://docs.influxdata.com/enterprise_influxdb/v1.8/administration/backup-and-restore/#restore-utility
> 
> > and any idea what I can do to change this from a docker container to a LXC to run with proxmox ?
> 
> I will try to give a solution on the new minor version v1.8.10-c1.2.0, which will be released next one or two months.

when will v1.8.10-c1.2.0 be release ?

**Comment by Jinyun-Liu** [comment]
expect!

**Comment by hoaihuongbk** [comment]
Same me, waiting for influxdb 2

**Comment by yunzhezyz** [comment]
is the plan is stop for open source? @chengshiwen

**Comment by chengshiwen** [comment]
> is the plan is stop for open source? @chengshiwen

The cluster plan for InfluxDB 1.x will continue to be open source and maintained. There is no plan for InfluxDB 2.x yet, so I will probably consider the cluster open source solution for 3.x directly.

**Comment by hungrybirder** [comment]
Waiting for open source solution for 3.x.

**Comment by flyingbeecd** [comment]
Waiting for open source solution for 3.x.

**Comment by MAngel666** [comment]
Waiting for open source solution for 3.x.
